### PR TITLE
adds introduction-jsx para of introducing-jsx page

### PR DESCRIPTION
**Pull request by arshadkazmi42**
Related to #1

### CODE REVIEW

**Comment by tesseralis** [comment]
@arshadkazmi42 is there a reason you're doing these one section at a time?

**Comment by arshadkazmi42** [comment]
@tesseralis We were doing this to make translation and review process easier. As some pages are huge and its becoming a bit complex while reviewing.
I have updated the issue also to show page section wise.

**Comment by tesseralis** [comment]
What I'd recommend, and what I've seen other languages do, is turn this into a "work in progress". You can work on one section at a time and update the PR as you go, asking people to review sections as you go by them.

I think it's important to keep a page on a single PR -- if it's split up into multiple sections, the style can get inconsistent and it'll be harder to read and verify. This was the problem we had with Crowdin: people only translated sections of even single sentences of a page and the end result would be a mishmash of English and the translated language.

**Comment by arshadkazmi42** [comment]
It does makes sense.
Let me update the issue to remove the sections, and we will keep 1 PR per page.

**Comment by saranshkataria** [comment]
I like the WIP idea and reviewing it in parts!

**Comment by tesseralis** [comment]
Yeah! And at the end you just need to skim it for consistency.

**Comment by saranshkataria** [comment]
Going back and forth is somewhat a challenge though. For example, #8 has 65 messages in the conversation tab and it is nowhere near done yet.

**Comment by saranshkataria** [comment]
Had a discussion with @arshadkazmi42 about the whole process. We will be having one PR for sections now onwards but to make the review process easier, we will go file by file and have a WIP PR as suggested by you @tesseralis. Thanks for the inputs. Will document it somewhere for other contributors soon.

**Comment by saranshkataria** [comment]
@arshadkazmi42 are we going to create a separate PR for this now? Or will you change the title for this and add subsequent changes section wise?

**Comment by arshadkazmi42** [comment]
I will be updating the same PR with subsequent section changes once current section is reviewed

File: content/docs/introducing-jsx.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 id: introducing-jsx
-title: Introducing JSX
+title: पेश है JSX
 permalink: docs/introducing-jsx.html
 prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+इस परिवर्तनशील घोषणा पर विचार करें::
```
Comment (saranshkataria) [inline content/docs/introducing-jsx.md]: परिवर्तनशील घोषणा -> too complex

File: content/docs/introducing-jsx.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 ```js
-const element = <h1>Hello, world!</h1>;
+const element = <h1>नमस्ते दुनिया!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+यह अजीब टैग सिंटैक्स न तो एक स्ट्रिंग है और न ही HTML है।
 
-It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with the full power of JavaScript.
+इसे JSX कहा जाता है, और यह जावास्क्रिप्ट के लिए एक सिंटैक्स एक्सटेंशन है। UI कैसा दिखना चाहिए, इसके लिए हम React के साथ इसका उपयोग करते हैं  JSX आपको एक टेम्पलेट भाषा की याद दिला सकता है, लेकिन यह जावास्क्रिप्ट की पूरी शक्ति के साथ आता है।
```
Comment (saranshkataria) [inline content/docs/introducing-jsx.md]: जावास्क्रिप्ट के लिए एक सिंटैक्स एक्सटेंशन -> javascript ka/ki syntex extension

Comment (saranshkataria) [reply content/docs/introducing-jsx.md]: रते हैं (insert full stop) JSX आपको

File: content/docs/introducing-jsx.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 
-JSX produces React "elements". We will explore rendering them to the DOM in the [next section](/docs/rendering-elements.html). Below, you can find the basics of JSX necessary to get you started.
+JSX React "एलिमेंट्स" का उत्पादन करता है। हम उन्हें [अगले भाग](/docs/rendering-elements.html) में DOM को प्रस्तुत करेंगे। नीचे, आप आरंभ करने के लिए JSX की मूल बातें पा सकते हैं।
```
Comment (saranshkataria) [inline content/docs/introducing-jsx.md]: हम उन्हें [अगले भाग](/docs/rendering-elements.html) में DOM को प्रस्तुत करेंगे -> changes the meaning of the line

File: content/docs/introducing-jsx.md (hunk 1 above)
Comment (arshadkazmi42) [reply content/docs/introducing-jsx.md]: how abot `वेरिएबल घोषणा` or `वेरिएबल डेक्लेरेशन` or should we not translate variable
`variable डेक्लेरेशन`

which one sounds better? Any suggestions?

File: content/docs/introducing-jsx.md (hunk 3 above)
Comment (arshadkazmi42) [reply content/docs/introducing-jsx.md]: How about this
`हम अगले भाग में उन्हें DOM में रेंडर करने के बारे में देखेंगे`

File: content/docs/introducing-jsx.md (hunk 1 above)
Comment (saranshkataria) [reply content/docs/introducing-jsx.md]: either one of `वेरिएबल घोषणा` or `वेरिएबल डेक्लेरेशन` looks good

File: content/docs/introducing-jsx.md (hunk 3 above)
Comment (saranshkataria) [reply content/docs/introducing-jsx.md]: cool

**Comment by saranshkataria** [comment]
Perfect!

**Comment by arshadkazmi42** [comment]
I will continue with next sections translation

**Comment by saranshkataria** [comment]
@arshadkazmi42 not reviewing the newer changes here, let's finish the other PR first 😄

File: content/docs/introducing-jsx.md [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
-Consider this variable declaration:
+इस वेरिएबल घोषणा पर विचार करें::
 
 ```js
-const element = <h1>Hello, world!</h1>;
+const element = <h1>नमस्ते दुनिया!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+यह अजीब टैग सिंटैक्स न तो एक स्ट्रिंग है और न ही HTML है।
```
Comment (IAmRC1) [inline content/docs/introducing-jsx.md:15]: Remove `hai` at the end!

**Comment by arshadkazmi42** [comment]
Closing due to inactivity

**Pull request closed without merging**